changelog.md

```diff
@@ -0,0 +1,86 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### [1.3.0] - [date]
```

Comment:
We don't need to add a version or `[date]` here. In the Keep a Changelog site they just list all changes under `[Unreleased]` as if it was a version number.